src/lib/helpers/print_helpers.rs

```diff
@@ -182,26 +182,34 @@ pub async fn print_market_details(
     println!("Quote vault key: {}", market_header.quote_params.vault_key);
 
     println!(
-        "Base Lot Size, in raw base units: {}",
+        "Raw base units per base lot: {}",
         get_decimal_string(
             market_metadata.base_atoms_per_base_lot,
             market_metadata.base_decimals
         ),
```

Comment:
The text says `raw base units per base lot` but the number to be printed is the `base_atoms_per_base_lot`?

Reply:
The actually printed value is a decimal string though